sandbox/helpers/nhs-number-helper.js

```diff
@@ -2,6 +2,40 @@ const Boom = require('boom')
 const patients = require('../services/patients')
 const nhsNumberValidator = require('../validators/nhs-number-validator')
 
+function _getPatient(nhsNumber) {
+    // Ideally should be done using options.validate object (https://hapi.dev/tutorials/validation/)
+    // But don't know how to customise the returned JSON when done this way
```

Comment:
could this comment be rephrased to 
Ideally should be done using options.validate object (https://hapi.dev/tutorials/validation/)
but the returned JSON cannot be customised

If it is something that you think is possible but did not want to spend more time investigating then raise a Tech Debt